README.md

```diff
@@ -1,78 +1,78 @@
-# Godot Engine
+# Redot Engine
 
 <p align="center">
-  <a href="https://godotengine.org">
-    <img src="logo_outlined.svg" width="400" alt="Godot Engine logo">
+  <a href="https://Redotengine.org">
```

Comment:
the redot link should be lowercase as stated by @Pierouge

Reply:
fixed, now I just gotta figure out how to squash my commits via online github lol